### DISCUSSION TITLE
程序自动被删

### DISCUSSION THREAD

**Discussion by ErgeAIA** [shuax/MouseInc.Settings#371] [General]
昨天更新了最新版,  今天莫名被删除两次, 装了火绒也没有记录, 啥情况啊, 没有任何提示, 就用着用着没反应, 一看文件没了

我是使用计划任务启动的程序

**Comment by ErgeAIA** [comment]
我去, 又没了, 回收站也没得, 直接消失

**Reply by luoyi2Roy** [reply]
windows安全中心干掉的?

**Reply by a1waysbeta** [reply]
可能性很大，WD设置里➕一个排除项

**Comment by sumeco** [comment]
我也是，莫名其妙的删除了。

**Reply by ErgeAIA** [reply]
昨天升级最新版, 到现在没事, v2.13.3

**Comment by Ctrl-Ling** [comment]
检查下windows defender
